src/common/io.py

```diff
@@ -2896,8 +2896,8 @@ def _get_sorted_vars(self):
             data_types = [
                 fmi.FMI_REAL,
                 fmi.FMI_INTEGER,
+                fmi.FMI_ENUMERATION,
```

Comment:
in fmi_util, we loop over `real`s, then `integer`s, then `bool`s. In FMI2 `enum`s are `int`s and thus need this ordering to be correctly bundled. 

Not super pretty, but we are not expecting to touch this again.